lawnotation-ui/server/trpc/routers/assignment.router.ts

```diff
@@ -53,7 +54,7 @@ export const assignmentRouter = router({
     .query(async ({ctx, input}) => {
       const serviceClient = ctx.getSupabaseServiceRoleClient();
 
-      const email_found = (await serviceClient.from('users').select('id').eq('email', input.email).maybeSingle());
+      const email_found = (await serviceClient.from('users').select('*').eq('email', input.email).maybeSingle());
```

Comment:
Change select('*') -> select() to keep it consistent with the rest of the codebase